x-pack/plugins/snapshot_restore/public/app/services/text/format_date.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
```

Comment:
we could define this in `x-pack/typings/@elastic/eui` to avoid having to ignore linting here